config/webpack.config.dev.js

```diff
@@ -16,7 +16,7 @@ var paths = require('./paths');
 module.exports = {
   devtool: 'eval',
   entry: [
-    require.resolve('webpack-dev-server/client') + '?http://localhost:3000',
```

Comment:
Haha seriously. This also fixes #194 in my testing.
I *think* this was only necessary because `contentBase` is configurable in Webpack.. which it isn’t here.